website/templates/emails/pending_retraction_non_admin.txt.mako

```diff
@@ -0,0 +1,7 @@
+Hello ${user.fullname},
+
+I just wanted to let you know, ${initiated_by} has requested a retraction for the following registration: ${registration_link}.
+
+Sincerely yours,
+
+The OSF Robots
```

Comment:
I just copied the signature used in the other emails. I'm happy to make them all singular if you'd like; either way I think they should all be the same.